File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int numsSize) {
        int freq[numsSize+1];
        memset(freq,0,sizeof(freq));
        for(int i=0;i<numsSize;i++)
        {
            freq[nums[i]]++;
        }
        for(int i=0;i<=numsSize;i++)
        {
            if(freq[i]==0)
            {
                return i;
            }
        }
        return -1;
}

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n=len(nums)
        freq=[0]*(n+1)
        for i in range(n):
            freq[nums[i]]+=1
        for i in range(n+1):
            if(freq[i]==0):
                return i
        return -1